src/templates/upload-dropdown-menu.html

```
@@ -46,21 +45,9 @@
-      <button type='button' class='dropdown-item' {% if not isVideo %}disabled{% endif %} data-action='insert-video-in-body' data-name='{{ upload.real_name|e('html_attr') }}' data-storage='{{ upload.storage }}' data-link='{{ upload.long_name|e('html_attr') }}' data-uploadid='{{ upload.id }}'><i class='fas fa-fw fa-video mr-1'></i>{{ 'Insert video'|trans }}</button>
-
-      {# INSERT AUDIO #}
-      {% set isAudio = ext matches '/(aac|flac|mp3|ogg|opus|wav)$/i' %}
-      <button type='button' class='dropdown-item' {% if not isAudio %}disabled{% endif %} data-action='insert-audio-in-body' data-name='{{ upload.real_name|e('html_attr') }}' data-storage='{{ upload.storage }}' data-link='{{ upload.long_name|e('html_attr') }}' data-uploadid='{{ upload.id }}'><i class='fas fa-fw fa-volume-low mr-1'></i>{{ 'Insert audio'|trans }}</button>
-
-      {# INSERT TEXT #}
-      {% set isTxt = ext matches '/txt$/i' %}
-      <button type='button' class='dropdown-item' {% if not isTxt %}disabled{% endif %} data-action='insert-plain-text' data-storage='{{ upload.storage }}' data-path='{{ upload.long_name|e('html_attr') }}'><i class='fas fa-fw fa-file-import mr-1'></i>{{ 'Insert text'|trans }}</button>
+      {# INSERT IN TEXT in edit mode #}
+      {% set isInsertable = ext matches '/(jpg|jpeg|png|gif|svg|bmp|mkv|mp4|ogv|webm|aac|flac|mp3|ogg|opus|wav|txt)$/i' %}
+      <button type='button' class='dropdown-item' {% if not isInsertable %}disabled{% endif %} data-action='insert-in-body' data-ext='{{ ext|e('html_attr') }}'data-name='{{ upload.real_name|e('html_attr') }}' data-storage='{{ upload.storage }}' data-link='{{ upload.long_name|e('html_attr') }}' data-uploadid='{{ upload.id }}'><i class='fas fa-fw fa-arrow-right-to-bracket mr-1'></i>{{ 'Insert in main text'|trans }}</button>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)